### PR TITLE
Adding unit test to check if login works with correct client Id

### PR DESCRIPTION
**Pull request by venomousboxer**
**Fixes** #8 

**changes :**
Added Espresso and UiAutomator dependencies and added a `LoginInstrumentedTest` to check if the login works with correct client ID

### CODE REVIEW

**Comment by venomousboxer** [comment]
@championswimmer please review this pull request.

File: app/build.gradle [hunk 1]
```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    buildToolsVersion "25.4.0"
     defaultConfig {
         applicationId "com.codingblocks.oauthandroid"
-        minSdkVersion 15
+        minSdkVersion 18
```
Comment (venomousboxer) [inline app/build.gradle]: I increased `minSdkVersion` to implement UiAutomator as it requires `minSdkVersion` 18 or above

**Comment by championswimmer** [comment]
travis still not building properly

**Comment by venomousboxer** [comment]
@championswimmer last commit is making travis build to fail should I revert the last commit

**Comment by championswimmer** [comment]
Yeah lets go without connectedCheck for now

**Comment by venomousboxer** [comment]
@championswimmer I have resolved the travis build issues can you please review this pull request now

**Pull request merged**